app/retail/templates/home/index.html

```diff
@@ -21,7 +21,7 @@
   <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
-    <meta name="description" content="The Gitcoin platform connects freelance developers with open bug bounties or online jobs, paid in crypto (ETH). Leverage a global workforce to quickly complete software development projects and coding jobs."/>
+    <meta name="description" content="The Gitcoin platform connects freelance developers with online jobs solving bug bounties, building features, designing creatives paid in crypto (ETH or any token). Leverage a global workforce to quickly complete software development and coding jobs, and find community funding for your project."/>
```

Comment:
should this read `and designing creatives`?

Reply:
fixed it